### PR TITLE
Profile page (show.html.erb) updated to show events you have RSVP'd along with the events you have created. 

### PR DESCRIPTION
**Pull request by brobs0111**
To test this: Use two different accounts that are email confirmed. Create events on both, and then RSVP to the other accounts events. Aside from that test everything else as you usually would.

### CODE REVIEW

**Comment by vinhnguyen200891337** [comment]
Everything is working fine and in order. The viewer is updated and easy to see.

**Comment by RogelioMart** [comment]
TL/DR: Found some issues with the website but those had nothing to do with Blake's Branch. Its good to merge.

Tested the code and found no problems with the branch. I did find some irregularities with the recurring events feature. Found out that the blocking feature although it does block a user it doesn't automatically kick them out of a conversation to kick someone out of a conversation it must be done through the jitsi chatroom. This is not an error but could be confusing to users, recommend putting a message that says tells the user to kick them out of the chatroom. I recommend someone look into these bugs.  But overall I found no errors. with the code that Blake wrote. It is good to merge.

**Comment by ece106** [comment]
The blocking bug is due to the site overload when querying the server every 3 sec, which was changed to 5 min. @honestabej & @asanchez0527 are working on it.

**Comment by ece106** [comment]
@RogelioMart you have a priority feature. Make a pull request into your test branch now or forever hold your peace.
